worlds/pokemon_emerald/CHANGELOG.md

```diff
@@ -2,7 +2,6 @@
 
 ### Features
 
-- Added many new item and location groups.
```

Comment:
Were these not added?

Reply:
It was merged the 29th, so it's not in a released version